client/src/components/Legend/index.tsx

```diff
@@ -14,7 +14,7 @@ import StoneFence from "../../assets/fences/stoneFence.png";
 import WoodenFence from "../../assets/fences/woodenFence.png";
 import styles from "./Legend.module.css";
 import ColoredFlag, { FlagType } from "../ColoredFlag";
-
+import Airplane from "../../assets/icons/airplane.png";
```

Comment:
I love how this Airplane is a helicopter 😂 You totally don't need to change it or anything, I just thought it was funny 🤣